containers/web/Dockerfile

```diff
@@ -5,7 +5,7 @@
 # *
 
 # Dockerfile for NodeJS application portion
-FROM node:latest
+FROM node:14
```

Comment:
I don't think we should include this change in the PR. It is something that needs more testing and will be in a separate PR. This also appears to be in PR #668.